pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -206,16 +207,25 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 		return r.reconcileAndError(instance, err)
 	}
 
-	if err = runCordonOrUncordon(r, node, true); err != nil {
+	if err = drain.RunCordonOrUncordon(r.drainer, node, true); err != nil {
 		return r.reconcileAndError(instance, err)
 	}
 
 	stop := make(chan struct{})
```

Comment:
revisit this stop channel if its needed , it was passed into the old drainer replaced now with the drain lib.

Reply:
deleted the channel in commit f4a5380497138abe83d95afea1abd1940f0e0e61